dbcon/joblist/rowestimator.cpp

```diff
@@ -380,7 +380,8 @@ float RowEstimator::estimateRowReturnFactor(const BRM::EMEntry& emEntry,
                     if (ct.colDataType == execplan::CalpontSystemCatalog::DECIMAL ||
                         ct.colDataType == execplan::CalpontSystemCatalog::UDECIMAL)
                     {
-                        uint128_t val = *(uint128_t*)msgDataPtr;
+                        uint128_t val;
```

Comment:
Plz take a look at the suggestion bellow in the tuple-bps.cpp regarding datatypes::TSInt128.

Reply:
Done.